components/dashboard/Sidebar/PostRow.tsx

```diff
@@ -30,29 +38,35 @@ const PostRow = ({ title, content, noteId, setShowSidebar }: PostRowProps) => {
 
   return (
     <div
-      className={`flex cursor-pointer flex-col gap-2 rounded-xl p-4 ${
-        selectedNoteId === noteId
-          ? "bg-slate-200 dark:bg-slate-700"
-          : "bg-slate-50 hover:bg-slate-100 dark:bg-slate-800 dark:hover:bg-slate-700"
-      }`}
-      onClick={() => switchNotesHandler(noteId)}
+      className={`flex items-center justify-between rounded-xl 
+    ${
```

Comment:
Removing the padding from the container causes inconsistent padding on the sides. The left is 1rem and right is 0.5rem. It should be 1rem (p-4) on all sides.